norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,318 @@
+    <choose>
+      <if type="article-journal">
+        <text variable="archive" text-case="capitalize-first" prefix="(Sitert fra " suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="accessed-date">
+    <choose>
+      <if type="personal_communication">
+        <date variable="accessed" prefix=", e-post, ">
+          <date-part name="day" form="numeric" suffix=". "/>
+          <date-part name="month" form="long" suffix=" "/>
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <date variable="accessed" prefix=" [Sitert " suffix="]">
+          <date-part name="day" form="numeric" suffix=". "/>
+          <date-part name="month" form="long" suffix=" "/>
+          <date-part name="year"/>
+        </date>
+      </else>
+    </choose>
+  </macro>
+  <macro name="locator">
+    <choose>
+      <if type="legislation bill" match="none">
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </if>
+      <else-if locator="section paragraph" match="any">
+        <text variable="locator" prefix=" § "/>
+      </else-if>
+      <else>
+        <group delimiter=" ">
+          <label variable="locator" form="short"/>
+          <text variable="locator"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <citation disambiguate-add-givenname="true" givenname-disambiguation-rule="primary-name" disambiguate-add-year-suffix="true" et-al-min="2" et-al-use-first="1">
+    <sort>
+      <key macro="author-short"/>
+      <key macro="title-short"/>
+      <key macro="issued"/>
+    </sort>
+    <layout delimiter=", ">
+      <group delimiter=" ">
+        <choose>
+          <if type="book thesis chapter article-journal article-newspaper article-magazine personal_communication" match="any">
+            <text macro="author-short"/>
+            <text macro="issued"/>
+          </if>
+          <else-if type="legal_case">
+            <text macro="title"/>
+          </else-if>
+          <else-if type="legislation">
+            <text macro="title-short"/>
+          </else-if>
+          <else-if type="report">
+            <text macro="author-short"/>
+            <text variable="number"/>
+          </else-if>
+          <else>
+            <text variable="title"/>
+          </else>
+        </choose>
+        <text macro="locator"/>
+      </group>
+    </layout>
+  </citation>
+  <bibliography entry-spacing="0">
+    <sort>
+      <key macro="type-sorting"/>
+      <key macro="author-full"/>
+      <key variable="issued"/>
+      <key variable="title"/>
+    </sort>
+    <layout>
+      <choose>
+        <if type="article-journal">
+          <group suffix=".">
+            <text macro="author-full"/>
+            <text prefix=". " macro="title"/>
+            <text prefix=", " font-style="italic" variable="container-title"/>
+            <text prefix=" Årg. " variable="volume"/>
```

Comment:
We have a preference for handling these as terms rather than hardcoded into prefix/suffix -- among other things, this would make localization easier, e.g. if there'd be a Nynorsk version or even an English one. Same for various other variables. So here this would be
```
<group delimiter=" ">
  <text term="volume" form="short"/>
  <text variable="volume"/>
</group>
```

Reply:
The updated/new style has actually simplified the journal article format so that the text term for "volume" is no longer used. I have updated the style to reflect this.

(The style is not yet fully compliant with the style guide. This was intended as an initial commit, which I will improve on over time.)